client/src/lib/firebase/auth.ts

```
@@ -34,6 +37,30 @@ export async function loginAnonymous() {
   return await signInAnonymously(auth);
 }
 
+export function deleteAccount() {
+  const user = auth.currentUser;
+
+  if (user !== null) {
```

we should probably throw an error in the else case